src/dev/storybook/aliases.ts

```diff
@@ -55,6 +55,7 @@ export const storybookAliases = {
     'x-pack/plugins/observability_solution/observability_ai_assistant/.storybook',
   observability_ai_assistant_app:
     'x-pack/plugins/observability_solution/observability_ai_assistant_app/.storybook',
+  observability_inventory: 'x-pack/plugins/observability_solution/inventory/.storybook',
```

Comment:
Relaying a comment from https://github.com/elastic/kibana/pull/195378:

This will need to be added to `.buildkite/scripts/steps/storybooks/build_and_upload.ts` if you want to preview it in the CI build